_includes/footer.html

```diff
@@ -61,13 +61,13 @@
         <div class="deepmost">
             <div class="copyright">&copy; {{ 'now' | date: "%Y" }} The ThingsBoard Authors</div>
             <div class="social">
-                <a href="https://github.com/thingsboard/thingsboard" class="github"></a>
-                <a href="https://app.gitter.im/#/room/#thingsboard_chat:gitter.im" class="gitter"></a>
-                <a href="https://www.youtube.com/thingsboard" class="youtube"></a>
-                <a href="https://stackoverflow.com/questions/tagged/thingsboard" class="stack-overflow"></a>
-                <a href="https://groups.google.com/forum/#!forum/thingsboard" class="mailing-list"></a>
-                <a href="https://www.facebook.com/thingsboard" class="facebook"></a>
-                <a href="https://twitter.com/thingsboard" class="twitter"></a>
+                <a href="https://github.com/thingsboard/thingsboard" class="github" aria-label="Link to our "></a>
+                <a href="https://app.gitter.im/#/room/#thingsboard_chat:gitter.im" class="gitter" aria-label="Link to our gitter"></a>
+                <a href="https://www.youtube.com/thingsboard" class="youtube" aria-label="Link to our youtube channel"></a>
```

Comment:
All product names should be in Capital: GitHub, Youtube, Google, Youtube, Twitter, Facebook, Stack Overflow, Gitter.